src/scripts/main.js

```diff
@@ -1,3 +1,417 @@
+
+label2.textContent = 'Position: ';
+label2.appendChild(positionForForm);
+
+const label3 = document.createElement('label');
+
+label3.textContent = 'Age: ';
+label3.appendChild(ageForForm);
+
+const label4 = document.createElement('label');
+
+label4.textContent = 'Salary ';
+label4.appendChild(salaryForForm);
+
+const labelOffice = document.createElement('label');
+
+labelOffice.textContent = 'Office: ';
+labelOffice.appendChild(officeForForm);
+
+form.appendChild(label1);
+form.appendChild(label2);
+form.appendChild(labelOffice);
+form.appendChild(label3);
+form.appendChild(label4);
+form.appendChild(submit);
+
+const pushNotification = (posTop, posRight, title, description, type) => {
+  const notification = document.createElement('div');
+
+  document.body.appendChild(notification);
+  notification.classList.add('notification');
+  notification.dataset.qa = 'notification';
+
+  const h2 = document.createElement('h2');
+
+  h2.textContent = title;
+  notification.appendChild(h2);
+
+  const about = document.createElement('p');
+
+  about.textContent = description;
+  notification.appendChild(about);
+
+  if (type === 'warning') {
+    notification.classList.add('warning');
+  }
+
+  if (type === 'error') {
+    notification.classList.add('error');
+  }
+
+  if (type === 'success') {
+    notification.classList.add('success');
+  }
+
+  notification.style.position = 'absolute';
+  notification.style.top = `${posTop}px`;
+  notification.style.right = `${posRight}px`;
+
+  setTimeout(() => {
+    notification.remove();
+  }, 5000);
+};
+
+submit.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  const row = document.querySelector('tr').cloneNode(true);
+  const userName = nameForForm.value;
+
+  const userPosition = positionForForm.value;
+  const userOffice = officeForForm.value;
+  const userAge = ageForForm.value;
+  const salaryInput = salaryForForm.value.toString();
+  const sal = salaryInput.split('').reverse();
+  const formattedZarplatnya = [];
+
+  for (let i = sal.length - 1; i >= 0; i--) {
+    if (i % 3 === 2 && i !== sal.length - 1) {
+      formattedZarplatnya.push(',');
+    }
+    formattedZarplatnya.push(sal[i]);
+  }
+
+  const userSalary = `$${formattedZarplatnya.join('')}`;
+
+  if (!userAge || !userName || !userPosition || !userSalary) {
+    pushNotification(
+      450,
+      200,
+      'Увага',
+      'Заповніть всі поля!.\n ' + 'Нам потрібні всі вказані дані.',
+      'error',
+    );
+
+    return;
+  }
+
+  if (userName.length < 4) {
+    pushNotification(
+      450,
+      200,
+      'Увага',
+      "Невірно введене ім'я користувача.\n " +
+        'Довжина імені повинна бути як мінімум 4 символи.',
+      'error',
+    );
+
+    return;
+  }
+
+  if (userAge < 18 || userAge > 90) {
+    pushNotification(
+      450,
+      200,
+      'Увага',
+      'Невалідна вікова категорія.\n ' +
+        'Наші працівники можуть бути віком від 18 до 90 включно.',
+      'error',
+    );
```

Comment:
The age validation logic is correct, but ensure that the error message is clear and user-friendly.